src/opcodes.c

```diff
@@ -28,6 +29,16 @@ void setStatusByChar(unsigned char value, struct CPU *cpu) {
   cpu->status.negative = (value >> 7) & 0x1;
 }
 
+unsigned char getStatusByChar(struct CPU *cpu) {
+  return (cpu->status.negative << 7)
+    | (cpu->status.overflow << 6)
+    | (cpu->status.break_cmd << 4)
+    | (cpu->status.decimal << 3)
+    | (cpu->status.interrupt << 2)
+    | (cpu->status.zero << 1)
+    | (cpu->status.carry << 0);
+}
```

Comment:
This is so cool!

Reply:
Usually people store the flags in a char and do something similar to destruct it into binary flags, I find this easier to read since you need to store/read this in only two oocodes